tasks/47_string-expansion/string-expansion.js

```diff
@@ -0,0 +1,16 @@
+export function stringExpansion(string) {
+  const len = string.length;
```

Comment:
Unnecessary const "len".
Should be part of "for" structure.

This optimization is known as caching of array.length property but latest browser engines do not benefit from this practice. So, let's remove it.